### PR TITLE
17-05 ChakraCore servicing release

### PR DESCRIPTION
**Pull request by rajatd**
Fixes the following CVEs impacting ChakraCore:

CVE-2017-0229
CVE-2017-0223
CVE-2017-0224
CVE-2017-0252
CVE-2017-0230
CVE-2017-0234
CVE-2017-0235
CVE-2017-0236
CVE-2017-0228
CVE-2017-0238
CVE-2017-0266

### CODE REVIEW

**Comment by suwc** [comment]
:shipit:

**Comment by suwc** [comment]
LGTM in terms of 37688 and 37814.

File: test/utf8/rlexe.xml [hunk 1]
```diff
@@ -29,4 +29,10 @@
       <compile-flags>-forceserialized -oopjit-</compile-flags>
     </default>
   </test>
+  <test>
+    <default>
+      <files>bugGH2656.js</files>
+      <compile-flags>-args summary</compile-flags>
```
Comment (dilijev) [inline test/utf8/rlexe.xml]: Is `-endargs` needed?

**Comment by dilijev** [comment]
Re: fix for CVE-2017-0230
Fix LGTM
Tests LGTM (I confirm this also covers the case where unicode characters appear in the source before the definition of the class, since there are many test cases in the file.)
/cc @boingoing

**Comment by dilijev** [comment]
Various CI problems in Jenkins and VSO.

File: test/utf8/rlexe.xml (hunk 1 above)
Comment (rajatd) [inline test/utf8/rlexe.xml]: yep. this test is actually failing the CI.
/cc @boingoing

**Comment by rajatd** [comment]
@dotnet-bot windows arm_release windows x64_release windows x86_release

**Comment by rajatd** [comment]
@dotnet-bot test windows arm_release

**Comment by rajatd** [comment]
@dotnet-bot test Windows x64_release

**Comment by rajatd** [comment]
@dotnet-bot  test Windows x86_release

**Comment by dilijev** [comment]
@rajatd There's a prefast warning there.

**Comment by dilijev** [comment]
@rajatd Note: you can re-run multiple builds from a single comment. Just added a section to the Wiki to explain: https://github.com/Microsoft/ChakraCore/wiki/Jenkins-Build-Triggers#re-run-multiple-individual-builds

**Comment by akroshg** [comment]
LGTM

**Pull request merged**